components/teams-service/storage/conformance/conformance_test.go

```diff
@@ -152,88 +150,88 @@ func testGetTeamByName(ctx context.Context, t *testing.T, s storage.Storage) {
-	name := "Foobear"
-	_, err := s.GetTeamByName(ctx, name)
+	id := "foobear"
+	_, err := s.GetTeam(ctx, id)
 	assert.Equal(t, storage.ErrNotFound, err)
 }
 
 func testGetTeamsForUser(ctx context.Context, t *testing.T, s storage.Storage) {
-	name := "Team1 With User"
-	team, err := s.StoreTeam(ctx, name, name+" Description")
-	require.NoError(t, err, "setup: failed to create test team")
+	id := "team-1-with-user"
+	team, err := s.StoreTeam(ctx, id, id+" Name", []string{})
+	require.NoError(t, err, "setup: failed to create test team-1-with-user")
 
-	name2 := "Team 2 With User"
-	team2, err := s.StoreTeam(ctx, name2, name2+" Description")
-	require.NoError(t, err, "setup: failed to create test team 2")
+	id2 := "team-2-with-user"
+	team2, err := s.StoreTeam(ctx, id2, id2+" Name", []string{})
+	require.NoError(t, err, "setup: failed to create test team-2-with-user")
 
-	nameWithout := "Team Without User"
-	teamWithout, err := s.StoreTeam(ctx, nameWithout, nameWithout+" Description")
-	require.NoError(t, err, "setup: failed to create test team 2")
+	idWithout := "team-without-user"
+	_, err = s.StoreTeam(ctx, idWithout, idWithout+" Name", []string{})
+	require.NoError(t, err, "setup: failed to create test team-with-user")
 
 	initialUsers := []string{"user-for-team-fetch", "two", "three"}
 
-	team, err = s.AddUsers(ctx, team.ID, initialUsers)
-	require.NoError(t, err, "setup: failed to add users to test team 1")
-	team2, err = s.AddUsers(ctx, team2.ID, initialUsers)
-	require.NoError(t, err, "setup: failed to add users to test team 2")
-	teamWithout, err = s.AddUsers(ctx, teamWithout.ID, []string{"two", "three"})
-	require.NoError(t, err, "setup: failed to add users to test team without user")
+	team, err = s.AddUsers(ctx, id, initialUsers)
+	require.NoError(t, err, "setup: failed to add users to team-1-with-user")
+	team2, err = s.AddUsers(ctx, id2, initialUsers)
+	require.NoError(t, err, "setup: failed to add users to team-2-with-user")
+	_, err = s.AddUsers(ctx, idWithout, []string{"two", "three"})
+	require.NoError(t, err, "setup: failed to add users to test team-without-user")
 
 	returnedTeams, err := s.GetTeamsForUser(ctx, "user-for-team-fetch")
 	require.NoError(t, err)
 	expectedFetchedTeams := []storage.Team{team, team2}
-	ids := []uuid.UUID{returnedTeams[0].ID, returnedTeams[1].ID}
-	expectedIDs := []uuid.UUID{expectedFetchedTeams[0].ID, expectedFetchedTeams[1].ID}
+	ids := []string{returnedTeams[0].ID, returnedTeams[1].ID}
+	expectedIDs := []string{expectedFetchedTeams[0].ID, expectedFetchedTeams[1].ID}
 
 	assert.ElementsMatch(t, ids, expectedIDs) // Comparing the objects wholly will fail due to updated_at :(
 }
 
 func testPurgeUserMembership(ctx context.Context, t *testing.T, s storage.Storage) {
-	name := "Team1 With User"
-	team, err := s.StoreTeam(ctx, name, name+" Description")
-	require.NoError(t, err, "setup: failed to create test team")
+	id := "team-1-with-user"
+	team, err := s.StoreTeam(ctx, id, id+" Name", []string{})
```

Comment:
Not really significant, but several of the variables (team, team2, teamWithout) are assigned multiple times before being examined, so all the earlier (unused) assignments could be removed (i.e. just use `_`).